packages/ui-components/src/components/breadcrumb-item/breadcrumb-item.tsx

```diff
@@ -0,0 +1,57 @@
+import { Component, Host, h, Prop, EventEmitter, Event } from '@stencil/core';
+import { throttle } from 'lodash-es';
+import { IBreadcrumbItem, EBreadcrumbItemTarget } from './breadcrumb-item.types';
+
+@Component({
+	tag: 'kv-breadcrumb-item',
+	styleUrl: 'breadcrumb-item.scss',
+	shadow: true,
+})
+export class KvBreadcrumbItem {
+	/** (required) The text to display on the breadcrumb */
+	@Prop({ reflect: true }) label!: string;
+	/** (optional) The breadcrumb's link */
+	@Prop({ reflect: true }) href?: string;
+	/** (optional) The target of the link (only used if href is provided) */
+	@Prop({ reflect: true }) target?: EBreadcrumbItemTarget;
+	/** (optional) Sets this breadcrumb styling to be the active one (usually the last one) */
+	@Prop({ reflect: true }) active?: boolean;
+	/** (optional) The separator to use */
+	@Prop() separator?: string;
+
+	/** Emitted when the user clicks on the breadcrumb */
+	@Event() breadcrumbItemClick: EventEmitter<IBreadcrumbItem>;
+
+	connectedCallback() {
+		this.clickThrottler = throttle(() => this.onItemClick(), 300);
+	}
+
+	private clickThrottler: () => void;
+	private onItemClick = () => {
+		this.breadcrumbItemClick.emit({
+			label: this.label,
+			href: this.href,
+			target: this.target,
+			active: this.active
+		});
+	}
+
+	render() {
+		return (
+			<Host>
+				<li class={{
+					'breadcrumb-item': true,
+					'active': this.active
+				}}>
+					<a
+						href={this.href}
+						target={this.target}
+						onClick={this.clickThrottler}>
+						{this.label}
+					</a>
+				</li>
+				{ !this.active && this.separator && <span class="separator">{ this.separator }</span> }
```

Comment:
_Just a recommendation for the future_

Move this condition to variable and reuse it here. Something like `shouldShowSeparador`.
I think that `JSX` in general is a bit hard to read and we must try to keep it as simple as possible.

Reply:
Thanks for the suggestion, I'll keep it in mind. I have made some changes to remove the this.active as a conditional ;)